docs/source/pages/lightning.rst

```diff
@@ -32,12 +32,13 @@ The example below shows how to use a metric in your `LightningModule <https://py
             preds = self(x)
             ...
             # log step metric
-            self.log('train_acc_step', self.accuracy(preds, y))
```

Comment:
since `self.accuracy(preds, y)` returns a tensor, the metric instance won't reset automatically because lightning doesn't have access to the instance.